docs/index.md

```diff
@@ -5,6 +5,22 @@ title: Home
 ![Comic: Distributed Pair
 Programmers](images/comics/dpp_0.png){:class="center"}
 
+<!-- TODO: add short description of saros-->
```

Comment:
Added a TODO, because I am not sure how we want to design the first page.

If you take a look at other dev tools ([gradle](https://gradle.org/), [docker](https://www.docker.com/), [npm](https://www.npmjs.com/), [intellij](https://www.jetbrains.com/idea/), [eclipse](https://www.eclipse.org/ide/)), the index page is only for marketing.
  * very short description of the tool
  * highlight main feature
  * cute images

Probably we want an index page with another design than the other pages.